public/static/js/spawned_hosts.js

```diff
@@ -332,62 +334,60 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
       }
     };
 
+    initializeModal = function(modal, title, action) {
+      $scope.modalTitle = title;
+      modal.on('shown.bs.modal', function() {
+        $scope.modalOpen = true;
+      });
+
+      modal.on('hide.bs.modal', function() {
+        $scope.modalOpen = false;
+      });
+    }
+
+    onEnterPressed = function(action) {
```

Comment:
This variable is named deceptively - when a function starts with "on," that implies that it's a function called when a certain event happens. In this case, the function actually attaches an event handler, so I would name this something like "attachEnterHandler"

Reply:
Good point! Fixed.